.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-## Pull Request Guidelines
+<!--
+Checklist for submitting a Pull Request
+
+Please remove this comment block before submitting.
```

Comment:
How about "(If an expanded release note is not needed, this entire block may be deleted)"

Also you've got <!-- comments around the entire list.